Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -373,6 +376,103 @@ defineSuite([
         });
     });
 
+    function checkPointAndFeatureCounts(tileset, features, points) {
+        var stats = tileset._statistics;
+
+        expect(stats.numberOfFeaturesSelected).toEqual(0);
+        expect(stats.numberOfFeaturesLoaded).toEqual(0);
+        expect(stats.numberOfPointsSelected).toEqual(0);
+        expect(stats.numberOfPointsLoaded).toEqual(0);
+
+        return Cesium3DTilesTester.waitForTilesLoaded(scene, tileset).then(function() {
+
+            scene.renderForSpecs();
+            expect(stats.numberOfFeaturesSelected).toEqual(features);
+            expect(stats.numberOfFeaturesLoaded).toEqual(features);
+            expect(stats.numberOfPointsSelected).toEqual(points);
+            expect(stats.numberOfPointsLoaded).toEqual(points);
+
+            viewNothing();
+            scene.renderForSpecs();
+
+            expect(stats.numberOfFeaturesSelected).toEqual(0);
+            expect(stats.numberOfFeaturesLoaded).toEqual(features);
+            expect(stats.numberOfPointsSelected).toEqual(0);
+            expect(stats.numberOfPointsLoaded).toEqual(points);
+
+            tileset._trimTiles = true;
+
+            scene.renderForSpecs();
+
+            expect(stats.numberOfFeaturesSelected).toEqual(0);
+            expect(stats.numberOfFeaturesLoaded).toEqual(0);
+            expect(stats.numberOfPointsSelected).toEqual(0);
+            expect(stats.numberOfPointsLoaded).toEqual(0);
+        });
+    }
+
+    it('verify batched features statistics', function() {
+        var tileset = scene.primitives.add(new Cesium3DTileset({
+            url : withBatchTableUrl
+        }));
+
+        return checkPointAndFeatureCounts(tileset, 10, 0);
+    });
+
+    it('verify unbatched features statistics', function() {
+        var tileset = scene.primitives.add(new Cesium3DTileset({
+            url : withoutBatchTableUrl
+        }));
+
+        return checkPointAndFeatureCounts(tileset, 10, 0);
+    });
```

Comment:
A test with `withoutBatchTableUrl` isn't needed because the features are still batched, just no batch table. Maybe you want `'./Data/Cesium3DTiles/Batched/BatchedNoBatchIds/'` instead (aka no features).